nocom/src/Account.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './InputBox.css';
+
+function InputBox({ className, onClick }) {
```

Comment:
Rename to Account
also Account(props){

Reply:
> Rename to Account also Account(props){

Please await  my final completion before you review

I wanted my branch on the repo for you to see since you asked

Thanks